components/components_CSS/Avatar/avatar.css

```diff
@@ -0,0 +1,56 @@
+
+@import url("/root.css");
+
+.component-wrapper{
+    background-color: var(--wrapper-background-color);
+    margin: 1rem;
+    padding: 2rem;
+    display: flex;
+    align-items: flex-end;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 2rem;
+
+}
+
```

Comment:
The component wrapper class would be common in all components, so you should place this class somewhere else.

Reply:
Yeah placed it in root but forget to remove here